slp2mp4/dolphinrunner.py

```diff
@@ -61,6 +62,29 @@ def count_frames_completed(self):
         print("Rendered ",num_completed," frames")
         return num_completed
 
+    def prep_dolphin_settings(self):
+        # Determine efb_scale value from resolution field in config
+        if self.conf.resolution not in RESOLUTION_DICT:
+            print("WARNING: configured resolution is not valid, using 480p")
+            efb_scale = RESOLUTION_DICT["480p"]
+        else:
+            efb_scale = RESOLUTION_DICT[self.conf.resolution]
+
+        # Apply graphics settings
+        gfx_ini = os.path.join(self.user_dir, "Config", "GFX.ini")
+        gfx_ini_parser = configparser.ConfigParser()
+        gfx_ini_parser.optionxform = str
```

Comment:
Need this to be case-sensitive

Reply:
configparser uses a different type for it's elements I think? This sets them to be strings, which preserves the case-sensitivity of the file.